code/modules/multiz/movement.dm

```diff
@@ -227,7 +227,7 @@
 		if(!area2.has_gravity())
 			return
 
-	if(!istype(src, /mob/living/carbon/human))
+	if(!istype(src, /mob/living/carbon/human) && !istype(landing, /turf/space))
```

Comment:
Also add an istype for turf/simulated/open. You don't take fall damage then either.